File-Formats/PDF/Converting-HTML-to-PDF.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Converting HTML to PDF | Syncfusion
-description: Learn how to convert HTML to PDF using 3 different rendering engines (Blink, WebKit and IE) with various features like TOC, partial web page to PDF etc.
+description: Learn how to convert HTML to PDF using 3 different rendering engines (Blink and IE) with various features like TOC, partial web page to PDF etc.
```

Comment:
@Sri-hari-haran-g Need to change **3** different rendering engines to **2** different rendering engines